docs/user-guide/side-bar.md

```diff
@@ -75,6 +75,10 @@ In addition to the layers information, the following are provided by the *Identi
 
 <img src="../img/side-bar/hightlight-point1.gif" class="ms-docimage"/>
 
+* The **Edit** button <img src="../img/button/edit_button.jpg" class="ms-docbutton"/> allows the user to edit the record in the [Attribute Table](attributes-table.md) of the geometry selected on the map. 
```

Comment:
The **Edit** button <img src="../img/button/edit_button.jpg" class="ms-docbutton"/> allows the user to open the [Attribute Table](attributes-table.md) in edit mode showing only layers records corresponding to the clicked point on the map.